composer.json

```diff
@@ -42,13 +42,13 @@
     "laravel/tinker": "^2.6",
     "laravel/ui": "^3.3",
     "laravelcollective/html": "^6.2",
-    "lcobucci/clock": "1.2.0",
-    "lcobucci/jwt": "^3.4.5",
+    "lcobucci/clock": "1.2.0|2.0.0",
```

Comment:
Looking at the lockfile, the actual upgrade *is* performed to 2.0.0 

That talks about a backwards-compatibility break which we'll have to check out as per: https://github.com/lcobucci/clock/releases/tag/2.0.0

Reply:
We never actually use this directly; it's only used by dependencies. I suspect we should remove it and allow Composer to do its job.